infrastructure/main.tf

```
@@ -22,9 +22,39 @@ module "key-vault" {
   create_managed_identity = true
 }
 
+locals {
+  vaultName = "${var.product}-${var.env}"
+}
+
+data "azurerm_key_vault" "damages_key_vault" {
+  name = local.vaultName
+  resource_group_name = "${local.vaultName}"
```

quotes not required here
```suggestion
  resource_group_name = local.vaultName
```